### PR TITLE
Add script to update aws-node daemonset images

### PR DESCRIPTION
**Pull request by sushrk**
**What type of PR is this?**
feature

**Which issue does this PR fix**:
N/A

**What does this PR do / Why do we need it**:
PR adds a script to update the aws-node daemonset images to the specified images in env $AMAZON_K8S_CNI & $AMAZON_K8S_CNI_INIT. This can be useful to run tests on a specific image tag. 

**If an issue # is not available please add repro steps and logs from IPAMD/CNI showing the issue**:
N/A

**Testing done on this change**:
Script run:

./scripts/update-cni-images.sh 
Replacing images in aws-k8s-cni manifest with env $AMAZON_K8S_CNI and $AMAZON_K8S_CNI_INIT
Applying aws-k8s-cni.yaml manifest to aws-node daemonset
serviceaccount/aws-node unchanged
customresourcedefinition.apiextensions.k8s.io/eniconfigs.crd.k8s.amazonaws.com configured
clusterrole.rbac.authorization.k8s.io/aws-node unchanged
clusterrolebinding.rbac.authorization.k8s.io/aws-node unchanged
daemonset.apps/aws-node configured
Waiting for daemon set "aws-node" rollout to finish: 0 out of 2 new pods have been updated...
Waiting for daemon set "aws-node" rollout to finish: 0 out of 2 new pods have been updated...
Waiting for daemon set "aws-node" rollout to finish: 0 out of 2 new pods have been updated...
Waiting for daemon set "aws-node" rollout to finish: 1 out of 2 new pods have been updated...
Waiting for daemon set "aws-node" rollout to finish: 0 of 2 updated pods are available...
Waiting for daemon set "aws-node" rollout to finish: 1 of 2 updated pods are available...
daemon set "aws-node" successfully rolled out

**Automation added to e2e**:
N/A

**Will this PR introduce any new dependencies?**:
N/A

**Will this break upgrades or downgrades. Has updating a running cluster been tested?**:
No.

**Does this change require updates to the CNI daemonset config files to work?**:
No

**Does this PR introduce any user-facing change?**:
No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: scripts/update-cni-images.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+source "$SCRIPTS_DIR/lib/k8s.sh"
+
+CONFIG_DIR="$SCRIPTS_DIR/../config"
+
+echo "Applying config/master/aws-k8s-cni.yaml manifest to aws-node ds"
+kubectl apply -f $CONFIG_DIR/master/aws-k8s-cni.yaml
+
+echo "Setting aws-node ds for testing"
+kubectl set image ds aws-node -n kube-system aws-node=$AMAZON_K8S_CNI aws-vpc-cni-init=$AMAZON_K8S_CNI_INIT
```
Comment (vikasmb) [inline scripts/update-cni-images.sh:16]: Is there a particular reason for  2 step update ? Instead, can we replace the image tag in yaml file prior to applying it ? Tag replacement approach is followed in https://github.com/aws/amazon-vpc-cni-k8s/blob/master/scripts/run-integration-tests.sh#L194

Comment (sushrk) [reply scripts/update-cni-images.sh:16]: Thanks for the reference! Replacement in the yaml file before applying the manifest should work too, I'll update the PR.

**Review by vikasmb — APPROVED** [review]
lgtm

File: scripts/lib/k8s.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# until the timeout. Namespace and timeout are optional parameters
+check_ds_rollout() {
+    local __ds_name=${1:-}
+    local __namespace=${2:-}
+    local __timeout=${3:-"2m"}
+    local __args=""
+    if [ -n "$__namespace" ]; then
+        __args="$__args-n $__namespace"
+    fi
+    kubectl rollout status ds/"$__ds_name" $__args --timeout=$__timeout
```
Comment (jayanthvn) [inline scripts/lib/k8s.sh:13]: Can we add an error if rollout fails?

Comment (sushrk) [reply scripts/lib/k8s.sh:13]: kubectl will print error if timed out, eg: error: timed out waiting for the condition

Comment (jayanthvn) [reply scripts/lib/k8s.sh:13]: Sounds good.

**Review by jayanthvn — APPROVED** [review]

**Pull request merged**